src/Hooks/CommandStartingListener.php

```diff
@@ -18,14 +18,15 @@
 use Laravel\Nightwatch\Core;
 use Laravel\Nightwatch\Facades\Nightwatch;
 use Laravel\Nightwatch\State\CommandState;
-use Laravel\Nightwatch\Types\Str;
 use Throwable;
 
 /**
  * @internal
  */
 final class CommandStartingListener
 {
+    private bool $hasRun = false;
+
```

Comment:
We were previously using the state to work out if this had already run. I would prefer we extract this bool to keep track if it has already run.

I feel this is clearer as well.